src/SDK.ts

```diff
@@ -362,10 +393,18 @@ export default class SDK implements ISDK {
         // Resolves only if it contains reconnectable chats response which only happens on status 200
         if (data) {
           if (this.logger) {
-            this.logger.log(LogLevel.INFO,
-              OCSDKTelemetryEvent.GETRECONNECTABLECHATS,
-              { Region: response.data.Region, ElapsedTimeInMilliseconds: elapsedTimeInMilliseconds, TransactionId: response.headers[Constants.transactionid] },
-              "Get Reconnectable Chats Succeeded and old session returned");
+
+            const customData = {
```

Comment:
Isn't it missing `requestId`?

Reply:
getReconnectableChats does not have requestId